apps/vite/src/components/DropDown.tsx

```
@@ -26,27 +31,38 @@ const DropDown = ({ selected, setSelected, items, className }: DropDownProps) =>
           </Listbox.Button>
           <Transition
             as={Fragment}
             leave="transition ease-in duration-100"
             leaveFrom="opacity-100"
-            leaveTo="opacity-0">
+            leaveTo="opacity-0"
+          >
             <Listbox.Options className="scrollbar-hide absolute z-50 mt-1 max-h-60 w-auto overflow-auto rounded-md border bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none dark:border-gray-600 dark:bg-gray-800 sm:text-sm">
               {formattedItems.map((item, idx) => (
                 <Listbox.Option
                   key={idx}
```

Biome

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)